### ISSUE TITLE
`EmptyDataError: No Columns to parse from file` from `get_group_observations("FX_RATES_DAILY")`

### ISSUE THREAD

**Issue by tylercroberts** [tylercroberts/pyvalet#7]
**Describe the bug**
When trying to use the `get_group_observations()` method for the 'FX_RATES_DAILY' series, I get EmptyDataError, but this should not be the case.

**To Reproduce**
```
from pyvalet import ValetInterpreter
vi = ValetInterpreter()
df_group = vi.get_group_observations("FX_RATES_DAILY", response_format='csv')
```

**Expected behavior**
Return df containing all of the series in this group, perhaps with a column indicating which series it comes from.

**Screenshots**

**Other Details**
Happened on Python 3.7

**Comment by tylercroberts** [comment]
May also want to look into adding additional detail to the BoCException so we can see what URL actually failed.

**Comment by tylercroberts** [comment]
Both resolved in 9290ea8d36cd522e6a8232da12a3aa2fb2c9aad6.

Interestingly, it was caused by an issue with the series details, rather than the group observations themselves.

BoCException previously had all info if a logger was passed. Updated so it _always_ at least returns the exception name.
